flytekit/types/file/file.py

```diff
@@ -25,9 +26,8 @@ def noop():
 T = typing.TypeVar("T")
 
 
-@dataclass_json
 @dataclass
-class FlyteFile(os.PathLike, typing.Generic[T]):
+class FlyteFile(os.PathLike, typing.Generic[T], DataClassJSONMixin):
     path: typing.Union[str, os.PathLike] = field(
```

Comment:
We still use `marshmallow` here. Are we able to use `mashumaro`.field or any equivalent

Reply:
Hi @pingsutw,
Can you be more specific about this? the local will pass all test.